android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -1551,6 +1635,35 @@ public void setNestedScrollEnabled(boolean nestedScrollEnabled) {
       this.nestedScrollEnabled = nestedScrollEnabled;
     }
 
+    public void setSource(ReadableMap source) {
+      this.source = source;
+    }
```

Comment:
Maybe worth a comment (either here or where `source` is declared) that this doesn't actually load the new source into the WebView, it's just for detecting changes?

Or perhaps worth considering if the logic for actually updating the WebView should move from `RNCWebViewManager`'s `setSource` to this function....?